src/libtriton/bindings/python/modules/tritonCallbacks.cpp

```diff
@@ -773,38 +792,57 @@ namespace triton {
 
                 Py_DECREF(args);
                 /********* End of lambda *********/
-              }, function));
+              }, func));
               break;
 
             case callbacks::GET_CONCRETE_REGISTER_VALUE:
-              triton::api.addCallback(callbacks::getConcreteRegisterValueCallback([function](triton::arch::Register& reg) {
+              triton::api.addCallback(callbacks::getConcreteRegisterValueCallback([func, function_self](triton::arch::Register& reg) {
                 /********* Lambda *********/
                 /* Create function args */
-                PyObject* args = triton::bindings::python::xPyTuple_New(1);
-                PyTuple_SetItem(args, 0, triton::bindings::python::PyRegister(reg));
+                PyObject* args = nullptr;
+                PyObject * pyReg = triton::bindings::python::PyRegister(reg);
+                if (function_self) {
+                  args = triton::bindings::python::xPyTuple_New(2);
+                  Py_INCREF(function_self);
```

Comment:
I guess if you incref before the call, you should decref after the call, no? Or maybe it is done by the callee?